doc/source/ray-observability/user-guides/debug-apps/optimize-performance.rst

```diff
@@ -62,19 +62,19 @@ as well as some known problems. If you encounter other problems, please
 Visualizing Tasks in the Ray Timeline
 -------------------------------------
 
-The most important tool is the timeline visualization tool. To visualize tasks
-in the Ray timeline, you can dump the timeline as a JSON file by running ``ray
-timeline`` from the command line or ``ray.timeline`` from the Python API.
+1. The :ref:`timeline API <ray-core-timeline>` is available from Ray Dashboard.
```

Comment:
Add a link to the dashboard page timeline?

Reply:
Done